tests/manage/monitoring/conftest.py

```diff
@@ -863,3 +865,101 @@ def workload_storageutilization_85p_cephfs(
         measurement_dir,
         tmp_path)
     return measured_op
+
+
+@pytest.fixture
+def measure_noobaa_exceed_bucket_quota(
+    measurement_dir,
+    request,
+    mcg_obj,
+    awscli_pod
+):
+    """
+    Create NooBaa bucket, set its capacity quota to 2GB and fill it with data.
+
+    Returns:
+        dict: Contains information about `start` and `stop` time for
+        corrupting Ceph Placement Group
+    """
+    bucket_name = create_unique_resource_name(
+        resource_description='bucket',
+        resource_type='s3'
+    )
+    bucket = S3Bucket(
+        mcg_obj,
+        bucket_name
+    )
+    mcg_obj.send_rpc_query(
+        'bucket_api',
+        'update_bucket',
+        {
+            'name': bucket_name,
+            'quota': {
+                'unit': 'GIGABYTE',
+                'size': 2
+            }
+        }
+    )
+
+    def teardown():
+        """
+        Delete test bucket.
+        """
+        bucket.delete()
+
+    request.addfinalizer(teardown)
+
+    def exceed_bucket_quota():
+        """
+        Upload 5 files with 500MB size into bucket that has quota set to 2GB.
+
+        Returns:
+            str: Name of utilized bucket
+        """
+        nonlocal mcg_obj
+        nonlocal bucket_name
+        nonlocal awscli_pod
+        # run_time of operation
+        run_time = 60 * 11
+        awscli_pod.exec_cmd_on_pod(
+            'dd if=/dev/zero of=/tmp/testfile bs=1M count=500'
```

Comment:
will the fact that the files are filled with zeros have an effect on the amount of data written to the bucket?

Reply:
I am not sure but I think that it shouldn't. Otherwise it would be bug.